tools/cldr-apps/js/src/esm/cldrSurvey.mjs

```diff
@@ -103,20 +104,16 @@ function getXpathMap() {
 /**
  * Is the keyboard or input widget 'busy'? i.e., it's a bad time to change the DOM
  *
- * @return true if window.getSelection().anchorNode.className contains "dijitInp" or "popover-content",
- *		 else false
- *
- * "popover-content" identifies the little input window, created using bootstrap, that appears when the
- * user clicks an add ("+") button. Added "popover-content" per https://unicode.org/cldr/trac/ticket/11265.
- *
- * Called only from CldrSurveyVettingLoader.js
+ * @return true if busy
  */
 function isInputBusy() {
-  if (!window.getSelection) {
-    return false;
+  if (cldrForum.isFormVisible()) {
+    return true;
```

Comment:
Adding this condition to isInputBusy is not elegant. It reflects the fact that we have Vue components (buttons) embedded in legacy DOM elements in the Info Panel, and the whole form is attached to one of those buttons and would get obliterated by an auto-scheduled page update if we didn't prevent it